source/edge-server/mongodb.txt

```diff
@@ -10,4 +10,100 @@ Edge Server MongoDB API Support
    :depth: 2
    :class: singlecol
 
-This is a placeholder page.
+Atlas Edge Server natively implements a subset of the MongoDB wire protocol,
+which allows you to connect to an App through one of its linked MongoDB 
+Atlas data sources using standard MongoDB drivers and tools. 
+
+Clients use a specialized MongoDB URI connection string to connect and 
+send requests.
+
+Edge Server currently supports a subset of the MongoDB APIs over the 
+wire protocol.
+
+.. _edge-server-wireprotocol-connect:
+
+Connect to the Edge Server
+--------------------------
+
+The wire protocol server is on by default. You can connect to it via the 
+host machine using this MongoDB URI:
+
+.. code:: console
+
+   mongodb://EDGE_SERVER_HOSTNAME:PORT
```

Comment:
just change to mongodb://EDGE_SERVER_HOSTNAME:27021

also should note that port 27021 needs to be made accessible to the client (via portmapping or firewall rules as appropriate)